lib/nn_model/train.py

```diff
@@ -77,7 +76,7 @@ def train_model(nn_model, w2v_model, tokenized_dialog_lines, index_to_token):
     start_time = time.time()
     sents_batch_iteration = 1
 
-    for full_data_pass_num in xrange(1, FULL_LEARN_ITER_NUM + 1):
+    for full_data_pass_num in range(1, FULL_LEARN_ITER_NUM + 1):
```

Comment:
Because xrange is deprecased in pythpn 3 and i use python 3
Dňa 16.3.2016 7:27 PM používateľ "Nicolas" <notifications@github.com>
napísal:

> In lib/nn_model/train.py
> <https://github.com/nicolas-ivanov/debug_seq2seq/pull/8#discussion_r56389227>
> :
>
> > @@ -77,7 +76,7 @@ def train_model(nn_model, w2v_model, tokenized_dialog_lines, index_to_token):
> >      start_time = time.time()
> >      sents_batch_iteration = 1
> >
> > -    for full_data_pass_num in xrange(1, FULL_LEARN_ITER_NUM + 1):
> > +    for full_data_pass_num in range(1, FULL_LEARN_ITER_NUM + 1):
>
> why to prefer range() over xrange() here?
> xrange() seems to be more computationally efficient
>
> http://stackoverflow.com/questions/94935/what-is-the-difference-between-range-and-xrange-functions-in-python-2-x
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nicolas-ivanov/debug_seq2seq/pull/8/files/5215c1cf85e88f4171280142adeb321c82c40db1#r56389227>
>